File: Backjoon/밤양갱.swift
/*
 밤양갱
 https://www.acmicpc.net/problem/31926
*/

let n = Int(readLine()!)!
var count = 0
while (1 << count) < n {
    count += 1
}
print(10 + count - ((1 << count) == n ? 0 : 1))

File: Backjoon/Python/밤양갱.py
"""
 밤양갱
 https://www.acmicpc.net/problem/31926
"""

n = int(input())
count = 0
while (1 << count) < n:
    count += 1
print(10 + count - (0 if (1 << count) == n else 1))
